src/main/java/com/ravlinko/concordion/extension/mockserver/tag/ResponseBodyTag.java

```diff
@@ -42,8 +42,12 @@ public void setUp(CommandCall commandCall, Evaluator evaluator, ResultRecorder r
 	@Override
 	public void verify(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
 		String body = (String) evaluator.getVariable(MOCK_RESPONSE_BODY_VARIABLE);
+		if (((String) evaluator.getVariable(MOCK_RESPONSE_BODY_VARIABLE)).equalsIgnoreCase("xml")) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the "xml" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1132)